appservice/src/connectToGitHub.ts

```diff
@@ -64,13 +67,18 @@ export async function connectToGitHub(node: AzureTreeItem, client: SiteClient, c
     };
 
     try {
-        const connectingToGithub: string = localize('ConnectingToGithub', '"{0}" is being connected to the GitHub repo. This may take several minutes...', client.fullName);
-        const connectedToGithub: string = localize('ConnectedToGithub', '"{0}" has been connected to the GitHub repo.', client.fullName);
-        await vscode.window.withProgress({ location: vscode.ProgressLocation.Notification, title: connectingToGithub }, async (): Promise<void> => {
-            ext.outputChannel.appendLine(connectingToGithub);
-            await client.updateSourceControl(siteSourceControl);
-            vscode.window.showInformationMessage(connectedToGithub);
-            ext.outputChannel.appendLine(connectedToGithub);
+        const deploying: string = localize('deploying', 'Deploying to "{0}"... Check output window for status.', client.fullName);
+        const deploymentComplete: string = localize('deploymentComplete', 'Deployment to "{0}" completed.', client.fullName);
+        const kuduClient: KuduClient = await getKuduClient(client);
+        await vscode.window.withProgress({ location: vscode.ProgressLocation.Notification, title: deploying }, async (): Promise<void> => {
+            ext.outputChannel.appendLine(deploying);
+            // tslint:disable-next-line:no-floating-promises
+            client.updateSourceControl(siteSourceControl).then(() => {
+                vscode.window.showInformationMessage(deploymentComplete);
+                ext.outputChannel.appendLine(deploymentComplete);
```

Comment:
Your messages are not showing at the right time. Deploy doesn't finish until waitForDeploymentToComplete

Reply:
This is the order that I'm seeing:

![image](https://user-images.githubusercontent.com/5290572/49104874-fc015600-f233-11e8-8349-1029b46c62de.png)

This makes sense to me.  It shows the kudu logs while the build is happening and then says the Deployment is completed when that is finished.

Reply:
A couple things:
1. Not sure we need to print a message that says "Check output window" to the output window 😂
1. Okay makes sense about the order, but I'd still prefer if you did it after `waitForDeploymentToComplete` to make it more clear in the code (rather than using `then`)